### ISSUE TITLE
No effect with `opacity` in `draw` operator 

### ISSUE THREAD

**Issue by Watson1978** [ImageMagick/ImageMagick#7301]
### ImageMagick version

7.1.1-29

### Operating system

Other (enter below)

### Operating system, version and so on

Linux 6.8.8, macOS 14.4.1

### Description

I think the `opacity` in `draw` operator should effect the image as same as when using `fill_opacity` and `stroke_opacity` in `draw` operator.
In fact, it is implemented as such.
https://github.com/ImageMagick/ImageMagick/blob/b9dc200cf91cb11044dae3cac2fa62054a243bcb/MagickCore/draw.c#L3228-L3253

However, it doesn't work with recently ImageMagick 6 and 7.

### Steps to Reproduce

```bash
~/imagemagick7.1/bin/convert -size 420x120 xc:gray \
  -draw "stroke 'blue'; fill 'yellow'; \
         stroke-width 10; \
         opacity  25%; roundrectangle  20,20, 100,100,5,5; \
         opacity  50%; roundrectangle 120,20, 200,100,5,5; \
         opacity  75%; roundrectangle 220,20, 300,100,5,5; \
         opacity 100%; roundrectangle 320,20, 400,100,5,5; " \
  result_7_1.png
```

If I ran above command with ImageMagick 6.8.9-10, it works as expected.

### Images

ImageMagick 6.8 | ImageMagick 6.9 | ImageMagick 7.1
-- | -- | --
<img src="https://github.com/ImageMagick/ImageMagick/assets/199156/d596f893-8c39-474f-afcb-747e8e556091"> | <img src="https://github.com/ImageMagick/ImageMagick/assets/199156/04cbec30-d9b2-4a07-9aa1-eb9c083ee211"> | <img src="https://github.com/ImageMagick/ImageMagick/assets/199156/c6beee5b-a39a-4b89-951a-66d489aca8af">

**Comment by fmw42** [comment]
I confirm, when I do your command on IM 7.1.1.32 Mac OSX Ventura, I get an image with an opaque alpha set on the result. But none of the drawn features show the specified opacity.

```
magick -size 420x120 xc:gray \
-draw "stroke blue fill yellow \
stroke-width 10 \
opacity  25% roundrectangle  20,20 100,100 5,5 \
opacity  50% roundrectangle 120,20 200,100 5,5 \
opacity  75% roundrectangle 220,20 300,100 5,5 \
opacity 100% roundrectangle 320,20 400,100 5,5" \
result_7_1.png

```

![result_7_1](https://github.com/ImageMagick/ImageMagick/assets/13458218/d2c72786-7c26-40a8-962e-ccfdcaa3e531)

**Comment by dlemstra** [comment]
I just pushed a patch to solve this issue in IM7. But we are getting a different (correct) output compared to 6.8:
![z](https://github.com/ImageMagick/ImageMagick/assets/10426229/67b35666-f6b2-45fc-b534-2b33c81be26c)

**Comment by fmw42** [comment]
Is that the IM 7 new result or the IM 6 result?  Can you show both?

**Comment by dlemstra** [comment]
I haven't found a proper fix for IM6 yet and this is the IM7 result.

**Comment by Watson1978** [comment]
Seems this issue has been solved at 6.9.13-11 and 7.1.1-33. Thanks.
